ui/pages/routes/routes.component.js

```diff
@@ -598,6 +603,23 @@ export default class Routes extends Component {
       isUnlocked &&
       !shouldShowSeedPhraseReminder;
 
+    let shouldShowLoadingPage = isLoading;
```

Comment:
nit: maybe `shouldShowLoadingPage` → `isLoadingShown`. I think since Loading isn't quite a page in our context we can omit it

Reply:
valid, I'll make that change